### PR TITLE
Mango v3 integration

### PR DESCRIPTION
**Pull request by microwavedcola1**
These adds instruction parsing and decoding for [mango-v3](https://mango.markets/). Pull request is currently in draft as I find and decode the last few instructions which are interesting.

### CODE REVIEW

**Comment by microwavedcola1** [comment]
The Travis CI issue seems unrelated 😅

File: explorer/src/components/instruction/MangoDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            mangoAccountKeyLocation={1}
+            title={"Withdraw"}
+            {...props}
+          />
+        );
+      case "InitSpotOpenOrders":
+        return (
+          <GenericMngoAccountDetailsCard
+            mangoAccountKeyLocation={1}
+            title={"InitSpotOpenOrders"}
```
Comment (SebastianBor) [inline explorer/src/components/instruction/MangoDetails.tsx:73]: nit: you could pass title in all these cases instead of repeating the string

**Review by SebastianBor — DISMISSED** [review]
Looks good, thx

File: explorer/src/components/instruction/MangoDetails.tsx (hunk 1 above)
Comment (microwavedcola1) [reply explorer/src/components/instruction/MangoDetails.tsx]: applied your suggestion in all generic cards

**Comment by mschneider** [comment]
Ran into rate limits when trying to load a bunch of transactions.
image

**Comment by mschneider** [comment]
LGTM

**Pull request merged**